ДЗ №5

AbellR3 opened the pull request:
Добавлены стили к текстовым элементам на главной странице

AbellR3 committed:
ДЗ №5

Добавлены стили к текстовым элементам

AbellR3 committed:
Добавлена сетка в header

AbellR3 committed:
Добавлена сетка в первую секцию главная станица

AbellR3 committed:
Наложена сетка на 2 секцию

Mikchail commented on style.css:
```diff
@@ -0,0 +1,198 @@
+/* Responsive */
+
+.responsive-title {
+    font-family: Montserrat;
```
если название шрифта состоит из 2 и более слов надо оборачивать в кавычки
и еще надо указывать веб-безопасный шрифт и тип шрифта указать

Mikchail reviewed (changes requested):
а где все остальные страницы ? 
и лучше создать папку pages и туда положить папки со страницами

AbellR3 committed:
добавлена сетка к 3 секции

AbellR3 committed:
добавлена сетка партнерам

AbellR3 committed:
Добавлена сетка на главную страницу

не добавлены фоновые заливка секции about-us и contacts

AbellR3 committed:
Исправление замечаний

Исправлены замечания. Шрифты обернуты в ковычки, добавлены веб безопасные шрифты

AbellR3 committed:
Нанесена сетка на страницу с контактами

Нанесена сетка на первую страницу и страницу с контактами
